src/opossum_lib/file_generation.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: 2023 TNG Technology Consulting GmbH <https://www.tngtech.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+import datetime
+import json
+from typing import Any, Dict, Union
+
+from networkx import DiGraph, weakly_connected_components
+
+from opossum_lib.helper_methods import _get_source_for_graph_traversal
+
+
+def write_dict_to_file(opossum_information: Dict, file_path: str) -> None:
+    with open(file_path, "w") as out:
```

Comment:
Optional: If we want to stick with pathlib, we could 
```suggestion
def write_dict_to_file(opossum_information: Dict, file_path: Path) -> None:
    with file_path.open("w") as out:
```

Reply:
done